src/UKHO.S100PermitService.StubService/Stubs/ProductKeyServiceStub.cs

```diff
@@ -8,21 +9,74 @@ namespace UKHO.S100PermitService.StubService.Stubs
 {
     public class ProductKeyServiceStub : IStub
     {
-        private readonly ProductKeyServiceConfiguration _productKeyServiceConfiguration;
+        public const string CONTENTTYPE = "Content-Type";
+        public const string APPLICATIONTYPE = "application/json";
+        private readonly ProductKeyServiceConfiguration productKeyServiceConfiguration;
 
         public ProductKeyServiceStub(ProductKeyServiceConfiguration productKeyServiceConfiguration)
         {
-            _productKeyServiceConfiguration = productKeyServiceConfiguration;
+            this.productKeyServiceConfiguration = productKeyServiceConfiguration;
```

Comment:
_productKeyServiceConfiguration = productKeyServiceConfiguration; <- is correct. Don't use 'this' here. Please amend the editor rule.

Reply:
Resolved